.bumpversion.cfg

```diff
@@ -0,0 +1,13 @@
+[bumpversion]
+current_version = 0.8.0-alpha
+commit = True
+tag = False
```

Comment:
Can't let this do tagging for us if we are using PRs for releases and want the tag to align with the ultimate `master` commit.